core/templates/dev/head/profile/profile.html

```diff
@@ -85,7 +85,8 @@ <h2 class="oppia-profile-username-small-screen">
         Interests:
         <span ng-if="subjectInterests.length > 0"><br>
           <!-- The following tag should remain on a single line due to http://bugzilla.mozilla.org/show_bug.cgi?id=922773. -->
-          <span ng-repeat="interest in subjectInterests track by $index"><span class="oppia-profile-subject-interest"><[interest]></span></span>
+          <span ng-repeat="interest in subjectInterests track by $index">
+               <span class="oppia-profile-subject-interest"><[interest]></span></span>
```

Comment:
This should be indented by only 2 spaces from the previous line, and the final </span> carried over to a line below, in order to preserve the convention that the opening and closing tags for the same HTML element should have the same indentation.